src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String NAME = "isoscelesTrapezoid";
+    private int heigth;
+    private int smallWidth;
+    private int bigWidth;
+
+    public IsoscelesTrapezoid(int heigth, int smallWidth, int bigWidth, String color) {
+        super(color);
+        this.heigth = heigth;
+        this.smallWidth = smallWidth;
+        this.bigWidth = bigWidth;
```

Comment:
your sides are random, how do you know which will be bigger? :)
please use more neutral naming